torch/_numpy/_casting_dicts.py

```diff
@@ -3,7 +3,7 @@
 import torch
 
 # These two dicts are autogenerated with autogen/gen_dtypes.py,
-# using numpy version 1.23.5.
+# using numpy version 1.24.3.
```

Comment:
@lezcano If I should re-run this with a particular version of numpy let me know. FWIW I made sure the diff only added entries none of the existing ones changed.